3lvia/kunde-elvia-web
# Feature/omega 187 show polygons if poweroutage is focused

OMEGA-187

**NB: Using `Record<string, any>` for map and maps from google maps api, because adding `@types/googlemaps` was not a trivial task. Created task OMEGA-249 to fix types**

## What is the reason for this change?

Show Polygon marking poweroutage, if focused.

### What is the expected result?

Show poweroutage polygon if focused.

## How is it solved?

Uses google maps api to create polygon and add to map

[Example screenshot of the solution if possible]
![image](https://user-images.githubusercontent.com/10927589/94897775-9bc34700-0490-11eb-94a5-f6abbcccc98b.png)


## Any changes/updates that may affect others?

|                       | Changed |
| --------------------- | ------- |
| Unit tests            |        |
| Dependencies          |         |
| Environment variables |         |
| Build scripts         |         |

## How can it be tested and verified?

Click on a chip and see if polygon shows (along with info box).
If closing info box, chip should show and polygon should hide.
